kong-docker/dbless/docker-compose.yaml

```
@@ -0,0 +1,26 @@
+version: "3.9"
+networks:
+  default:
+    name: kong-net
+services:
+  kong:
+    image: kong:2.5.0-alpine
+    hostname: kong
+    container_name: kong
+    environment:
+      KONG_DATABASE: 'off'
+      KONG_PROXY_ACCESS_LOG: '/dev/stdout'
+      KONG_ADMIN_ACCESS_LOG: '/dev/stdout'
+      KONG_PROXY_ERROR_LOG: '/dev/stderr'
+      KONG_ADMIN_ERROR_LOG: '/dev/stderr'
+      KONG_ADMIN_LISTEN: "0.0.0.0:8001, 0.0.0.0:8444 ssl"
+      KONG_DECLARATIVE_CONFIG: "/opt/kong/kong.yaml"
+    command: "kong start"
+    ports:
+      - "8000:8000"
+      - "8443:8443"
+      - "8001:8001"
+      - "8444:8444"
```

we try to encourage not opening the admin API port unprotected to the world at large. Doing it as follows makes it only open to localhost though this won't work if you're running the docker-compose somewhere remote obviously

```suggestion
      - "127.0.0.1:8001:8001/tcp"
      - "127.0.0.1:8444:8444/tcp"
```